astropy/time/core.py

```diff
@@ -10,7 +10,8 @@
 
 import copy
 import operator
-from datetime import datetime
+from datetime import datetime, date
+from time import strftime, strptime
```

Comment:
Seems that this be `datetime` instead of `time`.  The `time` object doesn't support fractional seconds and no `%f` format option.

Reply:
`datetime` doesnot support leap seconds. If we use time, I think we are presenting the users with a sensible choice. If you want to use `strftime` without `%f` but with leap seconds, use `Time.strftime`. If you want `%f` and not leap seconds, use `Time.datetime.strftime`

Reply:
Ugh, OK.  Then never mind the comments about using `datetime`.

Reply:
I edited my comments accordingly.